src/Avalonia.Base/PixelSize.cs

```diff
@@ -77,6 +78,27 @@ public static PixelSize Parse(string s)
             }
         }
 
+        /// <summary>
+        /// Try parsing <paramref name="source"/> as <see cref="PixelSize"/>.
+        /// </summary>
+        /// <param name="source">The <see cref="string"/> to parse.</param>
+        /// <param name="result">The result of parsing.</param>
+        /// <param name="separator">Optional parameter, identifies the separator the width and height, the default value is ','</param>
+        /// <returns><c>true</c> if <paramref name="source"/> is valid <see cref="PixelSize"/>, otherwise <c>false</c>.</returns>
+        public static bool TryParse(string source, [MaybeNullWhen(false)] out PixelSize? result, char separator = ',')
```

Comment:
`TryParse` methods usually accept `null`, the first argument should be changed to `[NotNullWhen(true)] string? source` and `source` checked for `null`.